modules/navigation.js

```diff
@@ -0,0 +1,42 @@
+// Sections
+const list = document.querySelector('#List');
+const add = document.querySelector('#New');
+const contact = document.querySelector('#Contact');
+
+// Navigation links array
+const links = document.querySelectorAll('#myLinks a');
+
+add.classList.add('hidden');
+contact.classList.add('hidden');
+
+// Function for showing the chosen section when a link gets clicked
+function GoToSection(e) {
+  // Name of the section to show
+  const name = e.target.textContent;
```

Comment:
* [ ]  👍 Your `GoToSection` function works perfectly, and that's great! As this project specifically requires the use of ES6 syntax, I would recommend utilizing arrow functions instead of regular functions. This practice aligns with the ES6 standard and enhances the readability and maintainability of the code. Please consider updating your code to use arrow functions wherever appropriate. Keep up the good work, and happy coding! 🚀🎉
 ## Please follow this requirement for all of your functions. 👍**